crypto/fipsmodule/ec/p384.c

```diff
@@ -438,39 +438,6 @@ static int ec_GFp_nistp384_cmp_x_coordinate(const EC_GROUP *group,
   return 0;
 }
 
-// ----------------------------------------------------------------------------
```

Comment:
Are these comments no longer true even in the `!EC_NISTP_USE_S2N_BIGNUM` case?

Reply:
explanation for `!EC_NISTP_USE_S2N_BIGNUM` is now in `ec_nistp.c` where the actual implementation is: https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/ec/ec_nistp.c#L390